lib/events.js

```diff
@@ -48,6 +48,13 @@ function lazyErrors() {
   return errors;
 }
 
+var util;
+function lazyUtil() {
```

Comment:
It must have a lazy load here because (https://github.com/nodejs/node/pull/15623):
> It is because events.js is loaded before all the other modules in the dependency tree (because of bootstrapping process. Lazy loading errors prevents errors from being loaded before events is loaded.